### PR TITLE
18 Stakeholders

### PR DESCRIPTION
**Pull request by nymann**
#18

### CODE REVIEW

**Comment by nymann** [comment]
I have added you as reviewer @Kechr since I can see that we may have some duplicate information regarding PO.

File: project_proposal/stakeholders.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### Stakeholders
+
+For this project we don't have any external stakeholders, which is a problem.
+When developing using SCRUM, there's a Product Owner role (abbreviated to PO
+from here on out). The PO should under normal circumstances be one from the
+customer's firm, and their job is to choose which features (user stories) that
```
Comment (Kechr) [inline project_proposal/stakeholders.md]: Remove apostrophe from customer's

File: project_proposal/stakeholders.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+### Stakeholders
+
```
Comment (Kechr) [inline project_proposal/stakeholders.md:2]: General comment: 
Avoid using abbreviations such as "don't", "there's" etc. to make it more formal.

**Review by Kechr — CHANGES_REQUESTED** [review]

File: project_proposal/stakeholders.md (hunk 1 above)
Comment (nymann) [reply project_proposal/stakeholders.md]: Removed

File: project_proposal/stakeholders.md (hunk 2 above)
Comment (nymann) [reply project_proposal/stakeholders.md:2]: I expanded all cases where I had shortened the words

Comment (Kechr) [inline project_proposal/stakeholders.md:2]: I'm not sure, but I think we should avoid using personal pronouns as well.

Comment (nymann) [reply project_proposal/stakeholders.md:2]: Personal pronouns, Kjako19?

Comment (Kechr) [reply project_proposal/stakeholders.md:2]: I, you, us, we, etc.

**Pull request merged**